app/views/devise/registrations/edit.html.slim

```diff
@@ -2,6 +2,16 @@ h2
   = t('.title', resource: resource_name.to_s.humanize)
 = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
   = devise_error_messages!
+  = f.label :avatar
+  - if current_user.avatar.url.present?
```

Comment:
https://robots.thoughtbot.com/rails-refactoring-example-introduce-null-object

Reply:
Take a look. Is it right?